libsel4vmmplatsupport/src/drivers/cross_vm_connection.c

```diff
@@ -151,59 +151,68 @@ static int reserve_event_bar(vm_t *vm, uintptr_t event_bar_address, struct conne
     return 0;
 }
 
-static vm_frame_t dataport_memory_iterator(uintptr_t addr, void *cookie)
+static vm_frame_t dataport_memory_iterator(uintptr_t frame_start, void *cookie)
```

Comment:
Based on discussion and my tentative conclusion in seL4/global-components#48, isn't a more proper name for the first argument `page_start`? In this function / file and everywhere else?

Reply:
Used `page_start` in this PR.